src/argilla/client/feedback.py

```diff
@@ -446,6 +446,43 @@ def add_records(
 
         Raises:
             ValueError: if the given records do not match the expected schema.
+
+        Examples:
+            >>> import argilla as rg
+            >>> rg.init(api_url="...", api_key="...")
+            >>> dataset = rg.FeedbackDataset(
+            ...     fields=[
+            ...         rg.TextField(name="text", required=True),
+            ...         rg.TextField(name="label", required=True),
```

Comment:
This is a comment for the future. Maybe we could separate free-text fields from term-like or class-like ones. Apart from semantics differences, for the second one you can improve the metrics or searches, since the search space can be reduced to a bunch of terms, instead of a full-text search.